relational/table.py

```diff
@@ -516,6 +520,27 @@ def _permit_authority_info(self, row, proj):
             row[self.index(self.PERMIT_AUTHORITY)] = ''
             row[self.index(self.PERMIT_AUTHORITY_ID)] = ''
 
+    def _bldg_permit_authority_info(self, row, proj):
+        permits = []
+        try:
+            fk_entries = proj.fields('permit_number',
+                                     PTS.NAME)
+            for (_, entries) in fk_entries.items():
+                for entry in entries:
+                    entry_latest = entry.get_latest('permit_number')
+                    if entry_latest:
+                        permit_number = entry_latest[0]
+                        permits.append(permit_number)
+
+            joined_permits = ','.join(permits)
+            if len(joined_permits) > 0:
+                row[self.index(self.BUILDING_PERMIT_AUTHORITY)] = \
+                    joined_permits
```

Comment:
the semantics of these are reversed from PERMIT_AUTHORITY/_ID - pls make them consistent

Reply:
Ahh good catch, can't believe I didn't catch this while I was writing code, done